msteams-platform/tabs/how-to/authentication/tab-sso-register-aad.md

```diff
@@ -0,0 +1,345 @@
+---
+title: Register your tab app with Azure AD
+description: Describes registering your tab app with Azure AD
+ms.topic: how-to
+ms.localizationpriority: medium
+keywords: teams authentication tabs Microsoft Azure Active Directory (Azure AD) access token SSO tenancy scope 
+---
+# Register your app in Azure AD
+
+Azure AD provides access to your tab app based on the app user's Teams identity. You'll need to register your tab app with Azure AD so that the app user who has signed into Teams can be given access to your tab app.
+
+## Enabling SSO on Azure AD
+
+Registering your tab app in Azure AD and enabling it for SSO requires making app configurations, such as generating app ID, defining API scope, and configuring the On-behalf-of (OBO) flow.
+
+:::image type="content" source="../../../assets/images/authentication/teams-sso-tabs/register-azure-ad.png" alt-text="Configure Azure AD to send access token to Teams Client app" border="false":::
+
+Create a new app registration in Azure AD, and expose its (web) API using scopes (permissions). Configure a trust relationship between the exposed API on Azure AD and your app. The OBO flow allows app users to access your tab app without any further need of consent. You can add client IDs for the trusted mobile, desktop, and web applications that you want to pre-authorize.
+
+You may also need to configure additional details, such as authenticating app users on the platform or device where you want to target your app. Azure AD configuration enables SSO for your tab app in Teams. It responds with an access token for validating the app user.
+
+User-level Graph API permissions are supported, that is, email, profile, offline_access, and OpenId. If you require access to additional Graph scopes, such as `User.Read` or `Mail.Read`, see [Get an access token with Graph permissions](tab-sso-graph-api.md).
+
+> [!NOTE]
+> The Microsoft Teams Toolkit registers the Azure AD application in an SSO project.
+
+### Before you register with Azure AD
+
+It's helpful if you learn about the configuration for registering your app on Azure AD beforehand. Ensure that you've prepared to configure the following details prior to registering your app:
+
+- **Single- or multi-tenant options**: Your app can be line-of-business (LOB) app, public app, or software-as-a-service (SaaS) application. The tenancy options may differ based on the type of your app and how you want to distribute it.
```

Comment:
Not sure if the terms "line-of-business (LOB) app, public app, or software-as-a-service (SaaS) application". Within Microsoft, LOB app is an app written by or for an enterprise, but outside of Microsoft it is an app that pertains to the customer's line of business, such as a medical or manufacturing app, regardless of who wrote it. SaaS is a hosting style that could be used within a single tenant or multiple tenants as an ISV would do. So I suggest rewording this to something like:

- **Single- or multi-tenant options**: Will your application be used in only the Microsoft 365 tenant where it is registered, or will many Microsoft 365 tenants use it? Applications written for one enterprise are typically single-tenant; applications written by an independent software vendor and used by many customers need to be multi-tenant so each customer's tenant can access the application.